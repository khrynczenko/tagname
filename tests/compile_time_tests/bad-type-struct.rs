use tagname::TagName;

#[derive(TagName)]
pub struct ArbitraryStruct{
}

fn main() {}
