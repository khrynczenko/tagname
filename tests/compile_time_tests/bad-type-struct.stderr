error: cannot derive `TagName` for struct types
 --> tests/compile_time_tests/bad-type-struct.rs:4:5
  |
4 | pub struct ArbitraryStruct {
  |     ^^^^^^
